Marlin/Configuration.h

```diff
@@ -748,7 +748,7 @@
  * Override with M92
  *                                      X, Y, Z, E0 [, E1[, E2...]]
  */
-#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 4000, 500 }
+#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 500 }
```

Comment:
It changed from `4000` to `200/1.25` with https://github.com/MarlinFirmware/Marlin/pull/15481, then back to `4000` with [b4d48fd](https://github.com/MarlinFirmware/Marlin//commit/b4d48fdc917cb89d89187c8b95b97cda9ae2c08d), so I think it was always `4000`.

Reply:
Most of the config is setup for an original Ultimaker, but it's also been updated over time to reflect current popular printer standards (like 1.75mm filament instead of 3.0mm, etc.).

Reply:
I would be more than happy with 400 as default.

Reply:
You should really be updating the config to match your specific hardware though.

Reply:
Was it for @Davidsg33k or me, but I agree with that one too.